wiki/Help_centre/Account_restrictions/vi.md

```diff
@@ -74,48 +68,58 @@ Nếu chúng tôi quyết định cho bạn quay lại, chúng tôi sẽ yêu c
 
+## Điều gì xảy ra nếu đây không phải lần đầu tôi khiếu nại?
+
+Với hầu hết trường hợp, mỗi lần tài khoản bạn bị hạn chế sẽ gấp đôi thời gian bạn bị hạn chế lần tiếp theo và cả thời gian chờ khiếu nại.
+
+Ví dụ, nếu bạn bị hạn chế do gian lận, quay lại sau khiếu nại, sau đó lại tiếp tục gian lận, bạn sẽ bị hạn chế tối thiểu 12 tháng (gấp đôi so với thời gian gốc là sáu tháng) trước khi chúng tôi xem xét bất kỳ khiếu nại nào từ bạn. Nếu bạn quay lại và lại gian lận tiếp, bạn sẽ bị hạn chế tối thiểu 24 tháng.
+
+Ghi chú thêm, việc quay lại sau lần vi phạm đầu tiên sẽ trở nên vô cùng khó khăn, vì chúng tôi phải cân nhắc những hành vi trong quá khứ của bạn và việc gỡ hạn chế cho bạn có đáng hay không. Rất ít (nếu có) người chơi trở về thành công sau lần hạn chế thứ ba.
+
 ## Email tôi bị bỏ qua / Tôi không nghĩ khiếu nại của tôi được giải quyết thoả đáng
 
 Trong trường hợp bạn nghĩ email của bạn bị đội ngũ hỗ trợ bỏ qua, xin hãy gửi cho chúng tôi một email tới [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh) chứa đầy đủ các thông tin dưới đây:
 
 - Bạn bị hạn chế bao lâu
 - Lý do chính bạn nghĩ mình bị hạn chế
 - Liệt kê các lần khiếu nại trong quá khứ bị bỏ qua/từ chối
-- Khái quát lý do tại sao bạn không hài lòng với kết quả của việc khiếu nại (nếu có thể)
+- Khái quát lý do tại sao bạn không hài lòng với kết quả của việc khiếu nại (nếu có)
 
-Mọi email gửi tới địa chỉ xem xét khiếu nại này sẽ được kiểm tra bởi một bên thứ ba không liên quan tới đội ngũ hỗ trợ thường ngày bạn gặp, được đánh dấu để điều tra kỹ lưỡng hơn.
+Mọi email gửi tới địa chỉ xem xét khiếu nại này sẽ được kiểm tra bởi một bên thứ ba không liên quan tới đội ngũ hỗ trợ thường ngày bạn gặp và được đánh dấu để điều tra kỹ lưỡng hơn.
 
-Mong bản hiểu rằng gửi email tới địa chỉ này chỉ đảm bảo bạn được xem xét lại khiếu nại, chứ không đảm bảo thay đổi kết quả gì nếu chúng tôi tìm ra bạn làm sai điều gì trong quá trình khiếu nại.
+Mong bạn hiểu rằng gửi email tới địa chỉ này chỉ đảm bảo bạn được xem xét lại khiếu nại, chứ không đảm bảo thay đổi kết quả nếu chúng tôi tìm ra bạn làm sai điều gì trong quá trình khiếu nại.
 
-**Chúng tôi chỉ phản hồi yêu cầu xem xét lại khiếu nại qua email gửi tới [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh).** Vui lòng không liên lạc tới người thuộc đội ngũ osu! (hoặc người thuộc đội tình nguyện) qua Twitter, Discord, hay bất cứ nền tảng nào khác nhằm lách qua điều khoản này - họ không giúp được bạn đâu.
+**Chúng tôi chỉ phản hồi yêu cầu xem xét/điều tra khiếu nại qua email gửi tới [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh).** Vui lòng không liên lạc với các thành viên của đội ngũ hỗ trợ (hoặc các thành viên của các đội tình nguyện) qua Twitter, Discord, hay bất cứ nền tảng nào khác nhằm lách qua điều khoản này - họ không giúp được bạn đâu.
```

Comment:
```suggestion
**Chúng tôi chỉ phản hồi yêu cầu xem xét/điều tra khiếu nại qua email gửi tới [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh).** Vui lòng không liên lạc với các thành viên của đội ngũ hỗ trợ (hoặc các thành viên của các đội tình nguyện) qua Twitter, Discord, hay bất cứ nền tảng nào khác nhằm lách qua điều khoản này - họ không thể giúp được bạn.
```

The original seems sarcastic.

Reply:
![](https://cdn.discordapp.com/emojis/930013006375571466.webp)